test-fastapi.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+set -x
+set -e
+
+if [ ! -d "./fastapi/" ]
+then
+    git clone https://github.com/tiangolo/fastapi.git
+fi
+cd fastapi
+git fetch --tags
+latest_tag_commit=$(git rev-list --tags --max-count=1)
+latest_tag=$(git describe --tags "${latest_tag_commit}")
+git checkout "${latest_tag}"
+pip install flit
+flit install
+SKIP_CYTHON=1 pip install -e ..
```

Comment:
maybe this can be achieved by changing the recipe to require `install` as a parent-recipe?

Reply:
I thought it wouldn't work, but it it did! :tada: